swan-lake/learn/tools-ides/setting-up-visual-studio-code/language-intelligence.md

```diff
@@ -52,11 +52,19 @@ These allow you to perform the below tasks easily based on the diagnostics and t
 - Add missing imports 
 - Create variable definitions
 - Create an undefined function
+- Map record types automatically
```

Comment:
```suggestion
- Map the record types automatically
```

Reply:
It is fine to say "Map record types" here as we aren't specifying a particular record type but referring to record types in general.